draft-packages/filter-menu-button/KaizenDraft/FilterMenuButton/FilterSplitButton.module.scss

```diff
@@ -45,7 +43,7 @@ $button-width: 383px;
   @include button-reset;
   @include button-base;
 
-  max-width: $button-width;
+  max-width: 500px;
```

Comment:
Curious about the context of this change from 383px to 500px.

Reply:
Hey, sorry I should have made this a draft first. I think the canary branch is behind master - these aren't actually my commits.

Reply:
Ah right! No worries